utilities/xmlrpcpp/test/TestValues.cpp

```diff
@@ -457,7 +457,7 @@ TEST(XmpRpc, errors) {
   EXPECT_FALSE(v);
 
   // Conversions to other types should now throw an XmlRpcException.
-  EXPECT_THROW((int)value, XmlRpcException);
+  EXPECT_THROW((void)(int)value; , XmlRpcException);
```

Comment:
Is a semicolon supposed to be here?

Reply:
Thanks for catching this. It is a left over of a previously tried workaround. I removed it.